amazon-kinesis-client/pom.xml

```diff
@@ -207,6 +207,10 @@
               <name>sqlite4java.library.path</name>
               <value>${sqlite4java.libpath}</value>
             </property>
+            <property>
+              <name>credentials</name>
+              <value>${credentials}</value>
```

Comment:
I will add the expectations of this parameter to the README. These credentials are the name of the IAM User that we want to run integration tests with. I will make it so that this property is optional. In the case that we want to override the default credentials on a system (local PC or within an AWS resource like CodePipeline), this parameter can be provided. Otherwise, the credentials installed in the test environment will be used.